packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -286,6 +286,17 @@ async function scanPullRequest(
   const owner = context.payload.repository.owner.login;
   const repo = context.payload.repository.name;
 
+  // If the head repo is null, we can not proceed.
+  if (
```

Comment:
It's nice that this is explicit, but in the future you can say `if (context.payload.pull_request.head.repo) { ...` and it will do a falsey check that looks for both null and undefined.

Reply:
I actually prefer exiting early because I'll have less indentation in the code.